### ISSUE TITLE
Case sensitive test failures

### ISSUE THREAD

**Issue by kieron-pivotal** [cloudfoundry/cf-acceptance-tests#382]
We're seeing several test failures where an app uses upper case characters, its route gets those characters lower-cased, and then a string comparison fails. E.g.

```
------------------------------
• Failure [108.365 seconds]
[v3]
/Users/development/go/src/github.com/cloudfoundry/cf-acceptance-tests/cats_suite_helpers/cats_suite_helpers.go:234
  v3 docker app lifecycle
  /Users/development/go/src/github.com/cloudfoundry/cf-acceptance-tests/cats_suite_helpers/cats_suite_helpers.go:240
    can run apps [It]
    /Users/development/go/src/github.com/cloudfoundry/cf-acceptance-tests/v3/app_lifecycle.go:208

    Timed out after 60.004s.
    Expected
        <string>: 404 Not Found: Requested route ('cats-1-app-9f4b7c3010137b30.nerdycats.garden-dev.cf-app.com') does not exist.

    to contain substring
        <string>: 404 Not Found: Requested route ('CATS-1-APP-9f4b7c3010137b30.nerdycats.garden-dev.cf-app.com') does not exist.


    /Users/development/go/src/github.com/cloudfoundry/cf-acceptance-tests/v3/app_lifecycle.go:249
------------------------------
```

We can make them pass by upper-casing the various names involved in the code, but are we missing something, as I can't see any other reports of this issue?

**Comment by davewalter** [comment]
Hi @kieron-pivotal,

We're not sure why this is failing for you. We checked the test and both the expected and actual strings that are being compared are generated by curling an app endpoint and receiving a 404 response from the router, so we don't understand why the case of the route in the two responses would be different. We added print statements and ran the test locally to confirm that, for us, the routes are lower-cased in both responses and so the test passes.

Is there possibly a change to your deployment that might cause this? You didn't mention which version of cf-deployment you are using, so it's hard for us to troubleshoot further. If nothing stands out, you might need to reach out to the Networking Program for assistance.

Sorry we can't be more help.
Dave and @weymanf
